Backpack-SwiftUI/CardList/Classes/BPKCardList.swift

```diff
@@ -0,0 +1,131 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import SwiftUI
+
+public struct BPKCardList<Element: Identifiable, Content: View>: View {
+    private let title: String
+    private let description: String?
+    private let layout: BPKCardListLayout
+    private let initiallyShownCards: Int
```

Comment:
nit: Do we want to include 'count' in the naming. So the naming is more precise?

Reply:
I thought that as well but was worried it could get too long
"initially" is a bit long though haven't found any good synonym

Reply:
`defaultVisibleCardCount` could be a contender, but naming things is **hard** 😅

Reply:
I never applied any change for this. I wouldn't mind changing it, but I also used that name because it's the one on the API table which I guessed other platforms would use.
Any strong opinions here? We could always rename it letter, but both options are fine with me 😄